### ISSUE TITLE
Incorrect auto fix，when i use the rule `@typescript-eslint/indent': ['error', 2]`

### ISSUE THREAD

**Issue by Junting-Liu** [eslint/eslint#12708]
**Tell us about your environment*

* **ESLint Version: ^5.16.0**
* **Node Version:v10.16.0**
* **npm Version:6.9.0**

**What parser (default, Babel-ESLint, etc.) are you using?  @typescript-eslint/parser**

**Please show your full configuration: **

```
module.exports = {
  root: true,
  env: {
    node: true
  },
  extends: ['plugin:vue/essential', '@vue/prettier', '@vue/typescript'],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    indent: 'off',
    '@typescript-eslint/indent': ['error', 2]
  },
  parserOptions: {
    sourceType: 'module',
    parser: '@typescript-eslint/parser'
  }
};

```

**What did you do? **

```js
  if(!item.satisfactionCount) {
      errors.push('xx');
  } else if(!numReg.test(item.satisfactionCount)
      || item.satisfactionCount - 100 > 0
      || item.satisfactionCount - 1 < 0) {
      errors.push('xx');
  }
```

```bash
eslint --fix
```

**after lint， the code was changed**
```js
if(!item.satisfactionCount) {
          errors.push('xx');
        } else if(!numReg.test(item.satisfactionCount)
                        || item.satisfactionCount - 100 > 0
          item.satisfactionCount - 1 < 0
          errors.push('xx');
        }
```

**What actually happened? **
after run eslint --fix ,the code  missed the      ){      in 5th line.

when i disable ` typescript-eslint/indent` , the code could be fixed corretly.

i want to kown why,and i want to use ` typescript-eslint/indent`  noramlly.

**Comment by kaicataldo** [comment]
The rule `@typescript-eslint/indent` is maintained by the `@typescript-eslint` team. Please create an issue in the [typescript-eslint](https://github.com/typescript-eslint/typescript-eslint) repository.
